### DISCUSSION TITLE
Latest Apple betas ... iOS 17 tvOS 17 etc.

### DISCUSSION THREAD

**Discussion by MSL-DA** [mikebrady/shairport-sync#1714] [Q&A]
Hi Mike ...

Do you have any idea why shairport-sync devices are showing "not responding"

It's not a huge problem for me as shairport-sync stops an ongoing playback when I leave my home. I'm just curious if this is something you are aware of?

**Comment by mikebrady** [comment]
Thanks for the question. I've been testing Shairport Sync with iPadOS 17 and macOS Sonoma betas and I get the feeling that there's a lot of activity in the domain of AirPlay infrastructure at the moment. But I haven't come across any new peculiarities.

Could you give us some details of your situation -- what are your source devices, apps and OS versions, what versions of Shairport Sync, etc? How does the system "know" that you've left home?

**Reply by dmitrygh** [reply]
iPad 16.3.1 doesnt see AirPlay... but older macOs 10.13.6 does, High Sierra
please help... iPad is the main player, thank you!

**Comment by MSL-DA** [comment]
> Thanks for the question. I've been testing Shairport Sync with iPadOS 17 and macOS Sonoma betas and I get the feeling that there's a lot of activity in the domain of AirPlay infrastructure at the moment. But I haven't come across any new peculiarities.
> 
> Could you give us some details of your situation -- what are your source devices, apps and OS versions, what versions of Shairport Sync, etc? How does the system "know" that you've left home?

I got my question wrong ... I mean "not responding" in the Home-app of course. Sorry

I have no problems with AirPlay from my devices (iPhone (iOS17), iPad (iPadOS17), Watch (watchOS10)) to Shairport-Sync

`4.2.1d0-54-gb965add1-AirPlay2-smi9-alac-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc`

**Reply by mikebrady** [reply]
Thanks for the clarification. Yes, this has also been happening to my devices with iOS 17 betas, but the HomePod mini is also “not responding”, so it’s hard to know what to make of it at the moment. Let’s hope it’s not permanent.

**Reply by MSL-DA** [reply]
That's not the case for me. I have multiple HomePods on beta version **17.0 (21J5339b)**

![IMG_0276](https://github.com/mikebrady/shairport-sync/assets/134940586/c4b7dd68-1248-4656-8002-59c13df345f7)

![IMG_0277](https://github.com/mikebrady/shairport-sync/assets/134940586/f567da01-7bf1-4ee7-aac2-352c529761b4)

![IMG_0278](https://github.com/mikebrady/shairport-sync/assets/134940586/263a4e89-d5af-4154-aaef-1f5d2e6b1aac)

![IMG_0279](https://github.com/mikebrady/shairport-sync/assets/134940586/58a3600a-520b-49d7-824e-1d8b9c76594e)

![IMG_0280](https://github.com/mikebrady/shairport-sync/assets/134940586/7cfc905e-6d98-4fb5-aab7-d415888877ac)

**Comment by mikebrady** [comment]
Thanks. Up to now, the only betas in my system were iPadOS 17 and macOS Sonoma -- haven't tried HomePod betas...

**[Update]** Sure enough, using the HomePod mini 17 beta works as @MSL-DA describes above. It only reinforces my view that there is a lot of development going on behind the scenes and it might be prudent to wait and see what comes out of all the changes when the final versions of iOS/iPadOS/HomePod OS eventually arrive.
